sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientSamples.cs

```diff
@@ -15,6 +15,7 @@ namespace Azure.Monitor.Query.Tests
     public class LogsQueryClientSamples: SamplesBase<MonitorQueryTestEnvironment>
     {
         [Test]
+        [Explicit]
```

Comment:
AzureActivity table is not always available causing tests to fail when running live. We have the functionality covered in live tests.